windows/src/desktop/kmshell/xml/splash.xsl

```diff
@@ -48,9 +48,7 @@
     <xsl:if test="count(//KeymanKeyboardInstalled[loaded]) = 0">No keyboards loaded - <a href="keyman:config">Open Configuration</a></xsl:if>
   </div>
 
-  <div id="version"><xsl:value-of select="/Keyman/Version"/>
-    <span id="beta"><xsl:value-of select="$locale/string[@name='S_BETA']"/></span>

```

Comment:
If `S_BETA` is going way, can it be removed from the strings.xml files?

Reply:
I think it's removed from kmshell/xml/strings.xml already and it will be automatically removed from the translated files when we import/export with Crowdin.